packages/core/src/cli/add-connector.ts

```diff
@@ -0,0 +1,25 @@
+import 'module-alias/register';
+import { getEnv } from '@silverhand/essentials';
+import chalk from 'chalk';
+
+import { addConnector } from '@/connectors/add-connectors';
+import { defaultConnectorDirectory } from '@/env-set';
+
+import { configDotEnv } from '../env-set/dot-env';
+
+configDotEnv();
+
+const addConnectorCli = async (packageName: string) => {
+  const connectorDirectory = getEnv('CONNECTOR_DIRECTORY', defaultConnectorDirectory);
```

Comment:
i think we can separate cli from core totally - no `envSet`, just raw env

Reply:
LOG-4051